Working Properly?

edengracee opened the pull request:
Can someone confirm mine is displaying correctly by downloading it now and letting me know? Thank you!

edengracee committed:
Update roommates.html

edengracee committed:
Update roommates.html

kmanthei committed:
Create dining.html

kmanthei committed:
Update index.html

changing name for hyperlink

kmanthei committed:
Update dining.html

kmanthei commented:
I am also wondering the same for my page Eden, I tried pushing to the GitHub desktop, and I couldn't figure out how to see the front end.

nyanford committed:
Update home.html

fixed hyperlinks

nyanford committed:
Update tamu.html

hyperlinked the files

nyanford committed:
Update roommates.html

nothing

nyanford committed:
Update tamu.html

added home hyperlink

nyanford committed:
Delete home

do not need it

nyanford committed:
Delete .github/TheSeniorExperience.github.io directory

about to reeupload as just files

nyanford committed:
Delete .github/workflows/blank.yml

not needed

nyanford committed:
Add files via upload

nyanford committed:
Update home.html

fixed all the hyperlinks

nyanford committed:
Update dining.html

fixed hyperlinks

nyanford committed:
Delete index.html

not needed

nyanford committed:
Update irene.html

removed photos and fixed hyperlinks

nyanford committed:
Update roommates.html

fixed hyperlinks

nyanford committed:
Update tamu.html

fixed hyperelinks

nyanford committed:
Update home.html

nyanford committed:
Update irene.html

labels

nyanford commented:
Hi, 

I tried downloading the file and ran into an error every time. I went through and fixed your hyperlinks with the correct labels and links. That was the only things I saw on my end

kmanthei committed:
Update roommates.html

Removed line 44 recommendations by just to see if this is causing the code not to run, I will replace once I see I is or isn't the problem. Just trouble shooting.

kmanthei committed:
Update roommates.html

Putting line 44 back because it wasn't the problem
